src/main/scala/uk/gov/nationalarchives/aws/utils/S3Utils.scala

```diff
@@ -0,0 +1,22 @@
+package uk.gov.nationalarchives.aws.utils
+
+import java.nio.file.{Path, Paths}
+
+import cats.effect.IO
+import monix.catnap.syntax._
+import software.amazon.awssdk.services.s3.S3AsyncClient
+import software.amazon.awssdk.services.s3.model.{GetObjectRequest, PutObjectRequest, PutObjectResponse}
+
+class S3Utils(client: S3AsyncClient) {
+  def upload(bucket: String, key: String, path: Path): IO[PutObjectResponse] = {
+    IO(client.putObject(PutObjectRequest.builder.bucket(bucket).key(key).build, path)).futureLift
+  }
+
+  def downloadFiles(bucket: String, key: String, path: Option[Path] = None) = {
```

Comment:
Maybe explicitly state the return type for this function for consistency: `IO[GetObjectResponse]`

Reply:
I've added this in. I've put some tests in for the s3 utils as well.